ocf_data_sampler/sample/uk_regional.py

```diff
@@ -0,0 +1,197 @@
+# uk_regional.py
+
+""" 
+PVNet - UK Regional sample / dataset implementation
+"""
+
+import numpy as np
+import pandas as pd
+import torch
+import matplotlib.pyplot as plt
+import logging
+
+from typing import Dict, Any, Union, List
+from pathlib import Path
+
+from ocf_data_sampler.numpy_sample import (
+    NWPSampleKey, 
+    GSPSampleKey,
+    SatelliteSampleKey
+)
+
+from ocf_data_sampler.sample.base import SampleBase
+
+
+logger = logging.getLogger(__name__)
+
+
+class PVNetSample(SampleBase):
+    """ Sample class specific to PVNet """
+    
+    # Feature space definitions
+    CORE_KEYS = {
+        'nwp',
+        GSPSampleKey.gsp,
+        GSPSampleKey.solar_azimuth,
+        GSPSampleKey.solar_elevation
```

Comment:
Would the GSP once be optional keys too? 
In live we wont need to load GSP at all